dts/bindings/dma/dma.yaml

```diff
@@ -0,0 +1,37 @@
+#
+# Copyright (c) 2019, Song Qiang <songqiang1304521@gmail.com>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+title: DMA Base Structure
+version: 0.1
+
+description: >
+    This binding gives the base structures for all DMA devices
+
+inherits:
+  !include base.yaml
+
+child:
+    bus: dma
+
+properties:
+    compatible:
+      type: string
+      category: required
+      description: compatible strings
+      generation: define
+
+    label:
```

Comment:
By including base.yaml, a lot of the duplication in here should go away.  Also we've removed 'generation:' field.